src/index.html

```diff
@@ -6,13 +6,65 @@
       name="viewport"
       content="width=device-width, initial-scale=1.0"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
     <title>Product cards</title>
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="product-card"
+      data-qa="card"
+    >
+      <img
+        src="images/imac.jpeg"
+        alt="Imac preview"
```

Comment:
alt attribute should describe the image content
<img width="456" alt="image" src="https://github.com/user-attachments/assets/1cbb2ca5-66cd-4586-b2ce-a5cec6422e1c">